cli/src/cli_util.rs

```diff
@@ -517,33 +517,33 @@ impl ReadonlyUserRepo {
     }
 }
 
-/// A branch that should be advanced to satisfy the "advance-branches" feature.
-/// This is a helper for `WorkspaceCommandTransaction`. It provides a type-safe
-/// way to separate the work of checking whether a branch can be advanced and
-/// actually advancing it. Advancing the branch never fails, but can't be done
-/// until the new `CommitId` is available. Splitting the work in this way also
-/// allows us to identify eligible branches without actually moving them and
-/// return config errors to the user early.
-pub struct AdvanceableBranch {
+/// A bookmark that should be advanced to satisfy the "advance-branches"
```

Comment:
done.

> > This is a pure mechanical change done via LSP, ripgrep and sed with some manual fixups.
> 
> I think this is obsolete, right?

Not really since I used them to do the bulk of changes

Reply:
The "pure mechanical change" makes it sound like you blindly replaced "branch" by "bookmark". I think you can just delete the sentence. If you want to keep it, maybe write it something like this:

> This patch was created mostly via LSP, ripgrep, and sed, with some manual changes to documentation etc. on top.

Reply:
I've rephrased the sentence, is that good now?

Reply:
Yep, sounds good, thanks!